mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -229,6 +237,106 @@ Two files will be created in the directory where the job was submitted:
 The `.o` file contains the output of the job.
 
 
+## vsc-venv: Python Virtual Environment Wrapper Script
+
+`vsc-venv` is a script that encapsulates the creation and management of Python virtual environments. 
+This avoids multiple issues with the default `venv` included in Python (`python -m venv`).
+
+
+One issue is that a virtual environment created for one cluster might not work on another. 
+Additionally, when activating a virtual environment, 
+the same centrally installed modules used during its creation must also be loaded.
+the `vsc-venv` command manages multiple virtual environments for different clusters in a transparent way, 
+while guaranteeing the same module environment.
+
+### Usage
+
+A virtual environment can be activated by running the following command:
+
+```bash
+$ module load vsc-env
+$ source vsc-venv --activate --requirements <requirements> [--modules module_file]
+```
+
+Here, `requirements` is the path to a file containing the python dependencies to install in the virtual environment.
+An optional `module_file` can be provided, which lists the modules to load before activating the virtual environment.
```

Comment:
````suggestion
$ source vsc-venv --activate --requirements requirements.txt [--modules modules.txt]
```

Here, `requirements.txt` is the path to a file containing the Python dependencies to install in the virtual environment.
The optional `--modules` option can be used to provide a `modules.txt` file that lists the modules to load before activating the virtual environment.
````